### PR TITLE
[W5.7][T13-3]Yingnan

### PR DESCRIPTION
**Pull request by NancyQuris**

### CODE REVIEW

File: src/seedu/addressbook/storage/StorageFile.java [hunk 1]
```diff
@@ -90,7 +90,10 @@ public void save(AddressBook addressBook) throws StorageOperationException {
      */
     public AddressBook load() throws StorageOperationException {
 
-        if (!Files.exists(path) || !Files.isRegularFile(path)) {
+        boolean fileExits = Files.exists(path);
```
Comment (davidchoo12) [inline src/seedu/addressbook/storage/StorageFile.java]: Typo `fileExits` should be `fileExists`

**Review by davidchoo12 — COMMENTED** [review]
Typo

**Comment by handshou** [comment]
Seen! Great job :)

**Comment by Gongjie663746** [comment]
Good job, Keep it up!

File: src/seedu/addressbook/commands/SortCommand.java [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+import seedu.addressbook.data.person.UniquePersonList;
+import seedu.addressbook.data.person.UniquePersonList.PersonNotFoundException;
+
+
+/**
+ * Sort the list of persons according to their name.
+ */
+public class SortCommand extends Command {
+
+    public static final String COMMAND_WORD = "sort";
```
Comment (parvathy-sudhir-pillai) [inline src/seedu/addressbook/commands/SortCommand.java:15]: Class variables should never be declared public by coding standard

File: src/seedu/addressbook/data/AddressBook.java [hunk 3]
```diff
@@ -61,6 +63,11 @@ public void clear() {
         allPersons.clear();
     }
 
+    /**
+     * Sort the list of persons according to their name in ascending order.
+     */
+    public List<Person> sortList() { return allPersons.sortList(); }
```
Comment (parvathy-sudhir-pillai) [inline src/seedu/addressbook/data/AddressBook.java:69]: Coding standard requires you to use K&R style brackets (aka Egyptian style) for writing method definitions.
The signature and definition shouldn't be on the same line.

File: src/seedu/addressbook/data/person/UniquePersonList.java [hunk 4]
```diff
@@ -1,11 +1,6 @@
 package seedu.addressbook.data.person;
 
-import java.util.ArrayList;
-import java.util.Arrays;
-import java.util.Collection;
-import java.util.Collections;
-import java.util.Iterator;
-import java.util.List;
+import java.util.*;
```
Comment (parvathy-sudhir-pillai) [inline src/seedu/addressbook/data/person/UniquePersonList.java:3]: Coding standard doesn't permit wildcard imports

**Review by parvathy-sudhir-pillai — APPROVED** [review]
Good effort on this PR. In particular, good job on keeping the PR relatively clean

Suggestions to improve further:
* update user and developer docs
* update I/O and jUnit tests to test the new functionality
* pay closer attention to the coding standard

Please close the PR after viewing comments.

File: src/seedu/addressbook/commands/SortCommand.java (hunk 2 above)
Comment (NancyQuris) [reply src/seedu/addressbook/commands/SortCommand.java]: Noted, thank you for the comment!

**Comment by NancyQuris** [comment]
I have noted the comments, thank you Ms Parvathy!

**Pull request closed without merging**